Misc/NEWS.d/next/Library/2020-05-02-14-24-48.bpo-40355.xTujaB.rst

```diff
@@ -0,0 +1,2 @@
+Fixed case where malformed :class:`ast.Dict` nodes could have keys or values
+thrown away by :func:`ast.literal_eval`. Patch by Curtis Bucher.
```

Comment:
Just a suggestion, not feeling very strongly about it but it took me a while to parse initially this NEWS entry:

```suggestion
Improve error reporting in :func:`ast.literal_eval` in the presence of malformed :class:`ast.Dict`
nodes instead of silently ignoring any non-conforming elements. Patch by Curtis Bucher.
```